website/docs/core-concepts/stacks/yaml-functions/include.mdx

```diff
@@ -170,3 +170,9 @@ On macOS and Linux:
     values: !include './values.yaml "<yq-expression-with-spaces>"'
     description: !include '"component description.md"'
 ```
+
+Sometimes for more complex cases you may need to utilize 3x double quotes, for examples:
```

Comment:
I rather we address the root cause. This is absurd. This is a hack, a workaround. This is not the solution.